pages/Book-Binder.md

```diff
@@ -0,0 +1,19 @@
+The Book Binder is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to combine different books together (much like an anvil would).  
+
+## Book Binder Settings
+By default, you cannot use this machine to get books above the vanilla limit.  
+However for server owners it is possible to adjust this and either add a custom limit, or no limit at all.
+
+| Name                     | Type    | Default Value |
+| ------------------------ | ------- | ------------- |
+| bypass-vanilla-max-level | boolean | false         |
```

Comment:
most settings names are pretty descriptive but think we need a description too?

Reply:
uh sure Ill just use a summary so it doesnt get too big